content/ru/edge-computing.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Периферийные вычисления
+status: Completed
+category: Technology
+---
+
+*Вычисления на периферии* (Edge computing), также известные как *пограничные* — это подход к [распределённым системам](/ru/distributed-systems/), при котором часть задач по хранению и обработке данных переносится от основного [центра обработки данных](/ru/data-center/) к источнику данных. 
+Собранные данные обрабатываются локально (например, на предприятии, в магазине или по всему городу), а не отправляются в централизованный дата-центр для обработки и анализа. 
+Эти локальные вычислительные устройства находятся на периферии системы, тогда как дата-центр является её центром. 
+Затем результаты, полученные на периферии, отправляются в главный ЦОД для дальнейшей обработки. 
+Показательными примерами подхода являются носимые гаджеты или компьютеры, которые следят за трафиком и анализируют транспортные потоки.
+
+## Какую проблему решает 
+
+В последнее десятилетие наблюдается рост числа периферийных устройств (таких как мобильные телефоны, умные часы и всевозможные датчики). 
+В некоторых случаях обработка данных в реальном времени не просто желательна, а жизненно необходима. 
+Достаточно вспомнить о самоуправляемых автомобилях. 
+Представьте, что данные с датчиков автомобиля приходилось бы передавать по сети в ЦОД для обработки, а затем обратно на автомобиль. 
+Подобная задержка в реакции, очевидно, неприемлема, когда дорога каждая секунда. 
+Хотя это доведенный до крайности пример, большинство пользователей не готовы ждать, когда их смартфон или иное умное устройство отреагирует на запрос, требуя мгновенной обратной связи.
+
+## Как именно решает проблему
+
+Как сказано выше, чтобы периферийные устройства были полезными, они должны выполнять хотя бы часть обработки и анализа локально, обеспечивая обратную связь почти в реальном времени. 
+Это достигается за счёт переноса части ресурсов для хранения и обработки данных из ЦОД к месту, где эти данные генерируются: периферийному устройству. 
+Обработанные и необработанные данные затем отправляются в ЦОД для дальнейшей обработки и хранения. 
+Одним словом, эффективность и скорость — главная движущая сила пограничных вычислений. 
```

Comment:
Perhaps my version is a bit less precise, but sounds better (IMHO) :sweat_smile: 
```suggestion
Одним словом, эффективность и скорость — решающие факторы для пограничных вычислений. 
```